src/main/java/it/pagopa/transactions/commands/data/UpdateAuthorizationStatusData.java

```diff
@@ -92,16 +90,7 @@ private void validateUpdateTransactionRequestValid(
                             requestValidationErrorHeader.formatted("psp id mismatch")
                     );
                 }
-                if (!pspTransactionId.equals(expectedPspTransactionId)) {
-                    logger.error(
-                            "Invalid redirect authorization outcome psp transaction id received. Expected: [{}], received: [{}]",
-                            expectedPspTransactionId,
-                            pspTransactionId
-                    );
-                    throw new InvalidRequestException(
-                            requestValidationErrorHeader.formatted("psp transaction id mismatch")
-                    );
-                }
+
```

Comment:
Why removing this check? This can still be done against baseTransaction authorizationRequestId, I would leave this coherence check instead of removing it altogether

Reply:
You are right, fixed in 408760f7